ocp_resources/ocs_initialization.py

```diff
@@ -0,0 +1,57 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class OCSInitialization(NamespacedResource):
+    """
+    OCSInitialization object.
+
+    API reference (source code linked here due to lack of API doc for this resource):
+        https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/ocsinitialization_types.go
+    """
+
+    api_group = NamespacedResource.ApiGroup.OCS_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        teardown=False,
+        privileged_client=None,
+        yaml_file=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        enable_ceph_tools=None,
+        **kwargs,
+    ):
+        """
+        enable_ceph_tools : Boolean
+            When set to True, creates a Deployment named "rook-ceph-tools" in the "openshift-storage" namespace.
```

Comment:
`in the "openshift-storage" namespace.`

I see that it can be on any namespace
See https://github.com/rook/rook/blob/master/deploy/examples/toolbox.yaml#L5
for example.

Reply:
It can, but by flipping this boolean, it's created in 'openshift-storage'. Similarly, the name can be different as well, but in this case it's "rook-ceph-tools".

Reply:
OK